go/fn/object.go

```diff
@@ -981,6 +981,15 @@ func (o *SubObject) GetSlice(k string) SliceSubObjects {
 	return o.NestedSliceOrDie(k)
 }
 
+// SetSlice set a slice of SubObject to the given fieldPath. It upserts the existing slice value.
```

Comment:
Since this function is for slice, "upserts" can be misread as "adding new subObjects and replace existing subObjects". 
Considering that users are mainly rely on this function description to learn our SDK, I suggest to be more specific and clear about what we does here. e.g. "SetSlice replaces the SliceSubObjects to the given field. It creates the field if not exists. If returns error if the field exists but not a slice type. "

@natasha41575  Do you have any recommendation to make the wording more concise?

Reply:
Your wording LGTM, I just made a minor suggestion below.